cypress/integration/files.spec.js

```diff
@@ -0,0 +1,294 @@
+    })
+  })
+
+  describe('Create file, delete file, edit file settings, and move files in file directories', () => {
+    before(() => {
+      cy.setCookie(COOKIE_NAME, COOKIE_VALUE)
+      window.localStorage.setItem('userId', 'test')
+
+      cy.visit(`/sites/${TEST_REPO_NAME}/documents`)
+      cy.contains('Create new directory').click()
+      cy.get("[id='file directory']").clear().type(DIRECTORY_TITLE)  
+      cy.get('button').contains(/^Create$/).click() 
+
+      // Assert
+      cy.contains(DIRECTORY_TITLE, { timeout: CUSTOM_TIMEOUT}).should('exist')
+    })
+
+    beforeEach(() => {
+      cy.setCookie(COOKIE_NAME, COOKIE_VALUE)
+      window.localStorage.setItem('userId', 'test')
+
+      cy.visit(`/sites/${TEST_REPO_NAME}/documents/${SLUGIFIED_DIRECTORY_TITLE}`)
+    })
+
+    it('Should be able to add file to file directory', () => {
+      // Set intercept to listen for POST request on server
+      cy.intercept({ method:'POST', url: `/v1/sites/${TEST_REPO_NAME}/documents` }).as('createFile')
+
+      cy.uploadMedia(FILE_TITLE, TEST_FILE_PATH)
+      // ASSERTS
+      cy.wait('@createFile') // should intercept POST request
+      cy.contains(FILE_TITLE, { timeout: CUSTOM_TIMEOUT}).should('exist') // file should be contained in Files
+    })
+
+    it('Should be able to edit an file in file directory', () => {      
+      cy.intercept({ method:'POST', url: `/v1/sites/e2e-test-repo/documents/${generateImageorFilePath(SLUGIFIED_DIRECTORY_TITLE, FILE_TITLE)}/rename/${generateImageorFilePath(SLUGIFIED_DIRECTORY_TITLE, OTHER_FILE_TITLE)}` }).as('renameFile')
+      
+      cy.renameMedia(FILE_TITLE, OTHER_FILE_TITLE)
+      // ASSERTS
+      cy.wait('@renameFile')
+      cy.contains(OTHER_FILE_TITLE, { timeout: CUSTOM_TIMEOUT}).should('exist') // File should be contained in Files
+
+      cy.intercept({ method:'POST', url: `/v1/sites/e2e-test-repo/documents/${generateImageorFilePath(SLUGIFIED_DIRECTORY_TITLE, OTHER_FILE_TITLE)}/rename/${generateImageorFilePath(SLUGIFIED_DIRECTORY_TITLE, FILE_TITLE)}` }).as('renameFileBack')
+      
+      cy.renameMedia(OTHER_FILE_TITLE, FILE_TITLE)
+      // ASSERTS
+      cy.wait('@renameFileBack')
+      cy.contains(FILE_TITLE, { timeout: CUSTOM_TIMEOUT}).should('exist')
+    })
+
+    it('Should be able to delete file from file directory', () => {
+      // Set intercept to listen for POST request on server
+      cy.intercept({ method:'DELETE', url: `/v1/sites/${TEST_REPO_NAME}/documents/${generateImageorFilePath(SLUGIFIED_DIRECTORY_TITLE, FILE_TITLE)}` }).as('deleteFile')
+
+      cy.deleteMedia(FILE_TITLE)
+      // ASSERTS
+      cy.wait('@deleteFile') // Should intercept DELETE request
+      cy.contains(FILE_TITLE, { timeout: CUSTOM_TIMEOUT}).should('not.exist') //File file name should not exist in Files
+    })
+
+    it('Should be able to move file from file directory to Files', () => {
```

Comment:
As discussed, these errors seem to be associated with running Cypress with the browser open. When running the tests in headless mode, these errors do not occur. We'll ignore these errors for now, since we have also confirmed that the file moving functionality works